README.md

```diff
@@ -57,6 +57,9 @@ VDK creates data processing workflows to:
     - A plugin framework that allows users to extend the framework according to their specific requirements.
 - Control Service: The Control Service allows users to create, deploy, manage, and execute data jobs in a Kubernetes runtime environment.
 
+A preview of the VDK CLI commands - `vdk --help`, `vdk create` and `vdk run`:
```

Comment:
What's the point of previewing the help command? Create and run are super valuable to have as a gif preview. Having help is a standard part of any CLI, so it doesn't seem special in that respect. Plus, whoever is reading this will not have time to read the help output and it's a bit annoying, imo.

Reply:
My idea was to show all the possibilities of VDK, but I don't want to put a separate image of the help command.
Also, gif is a catchy way to do it. The idea is not to show the help command, of course, but all the other commands that I don't show. But I can leave it out if you think it doesn't make sense.